### PR TITLE
Shadow for filter menu

### PR DESCRIPTION
**Pull request by KateVlaar**
image

### CODE REVIEW

File: app/src/main/res/layout/fragment_filter_menu.xml [hunk 1]
```diff
@@ -56,20 +56,35 @@
         android:layout_height="40dp"
         android:layout_marginTop="188dp"
         android:background="@color/zxing_transparent"
+        android:src="@drawable/filter_menu"
```
Comment (JawadHossain) [inline app/src/main/res/layout/fragment_filter_menu.xml:59]: is this being used, did you intend to add src to this button?

File: app/src/main/res/layout/fragment_filter_menu.xml [hunk 2]
```diff
@@ -56,20 +56,35 @@
...
+        android:layout_height="211.42424dp"
+        android:layout_marginTop="60dp"
+        android:background="@drawable/drop_shadow"
+        android:elevation="4dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.873"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+
```
Comment (JawadHossain) [inline app/src/main/res/layout/fragment_filter_menu.xml:76]: could remove blank line for consistency

**Review by JawadHossain — COMMENTED** [review]
Could also consider adding a stroke colour which would add border around the arrow above the filter menu.
Screenshot_1605228198
strokeAroundFilterMenu

File: app/src/main/res/layout/fragment_filter_menu.xml (hunk 1 above)
Comment (KateVlaar) [reply app/src/main/res/layout/fragment_filter_menu.xml]: Whoops, left in by mistake. Nice catch

File: app/src/main/res/layout/fragment_filter_menu.xml (hunk 2 above)
Comment (KateVlaar) [reply app/src/main/res/layout/fragment_filter_menu.xml:75]: Done

**Comment by KateVlaar** [comment]
> Could also consider adding a stroke colour which would add border around the arrow above the filter menu.
> Screenshot_1605228198
> strokeAroundFilterMenu

I think I like the shadow a bit better because it doesn't outline the arrow and fits in a bit better with the list items but this was a good suggestion

**Review by JawadHossain — APPROVED** [review]

**Review by IsaakC — APPROVED** [review]

**Pull request merged**